### PR TITLE
[website] much enhanced with nginx, esbuild and ssr 

### PR DESCRIPTION
**Pull request by phytonmk**
- After a long journey website got versioning and changelog notes.
- Now using esbuild and nginx: got rid of webpack, documentalist, nest and lot of other weird stuff.

### CODE REVIEW

File: tools/gcs-upload/index.ts [hunk 1]
```diff
@@ -2,6 +2,8 @@ import { Storage } from '@google-cloud/storage';
 import path from 'path';
 import fs from 'fs-extra';
 import { fileURLToPath } from 'url';
+import pLimit from 'p-limit';
```
Comment (s-kobets) [inline tools/gcs-upload/index.ts:5]: mmm, I think don't use new package there

Comment (phytonmk) [reply tools/gcs-upload/index.ts:5]: There are a lot of new dependencies in the pull request 😅

**Comment by s-kobets** [comment]
Very cool work website now, but I found some problem:
- page icons don't work (don't show icons)
- font apply very long time for all page (text jumps are visible)
- url `/terms/privacy/` is not available (problem: code don't find selector #1_...)
- emoji show strangely in the block `> ⚠️ This guide is for updated`

**Comment by phytonmk** [comment]
@s-kobets 

Fixed found issues.
Long fonts loading is a specific feature of dev server 🙃

**Pull request merged**